cmd/find-main.go

```diff
@@ -163,24 +162,44 @@ EXAMPLES:
 // checkFindSyntax - validate the passed arguments
 func checkFindSyntax(ctx *cli.Context) {
 	args := ctx.Args()
-
-	// help message on [mc][find]
-	if !args.Present() {
-		cli.ShowCommandHelpAndExit(ctx, "find", 1)
-	}
-
-	if ctx.Bool("watch") && !strings.Contains(args[0], "/") {
-		console.Println("Users must specify a bucket name for watch")
-		console.Fatalln()
+	if !ctx.Args().Present() {
+		args = []string{"./"}
 	}
-
-	// verify that there are no empty arguments
 	for _, arg := range args {
 		if strings.TrimSpace(arg) == "" {
 			fatalIf(errInvalidArgument().Trace(args...), "Unable to validate empty argument.")
 		}
 	}
 
+	// Extract input URLs.
+	URLs := ctx.Args()
+	for _, url := range URLs {
```

Comment:
comment like "Validate the extracted URLs. " will be good.

Reply:
Sure